### PR TITLE
Updated existing Administrator Onboarding Tasks

### PR DESCRIPTION
**Pull request by cwarnermm**
Overhauled existing Administrator Onboarding Tasks content to provide greater thought leadership and clearer deployment setup guidance and recommendations for System Admins.

### CODE REVIEW

**Comment by jasonblais** [comment]
@cwarnermm  While Neil is away, I'll cover for him on reviewing this PR for this week's go-live.

In terms of details, they're looking good! Just some high-level questions:

1 - If the intent is to make the admin feel like "going from zero to hero", I wonder how we can convey that on this page? I.e. instead of it feeling like a list of onboarding/setup tasks, how do I feel like a hero by the end of it.

For instance, instead of `Integrate with your directory services`, perhaps we can frame the section on how the admin can make the users lives easy by automating onboarding and account provisioning for them.

In other words, as an admin, I want to feel like after each step I'm becoming more of a hero after each step of the process.

2 - Propose adding a couple of screenshots to add a visual element, e.g. one for the workspace insights.

File: source/getting-started/admin-onboarding-tasks.rst [hunk 1]
```diff
@@ -13,146 +13,202 @@ Administrator Onboarding Tasks
...
-User Experience Optimizations
------------------------------
+Control product access with permissions
+----------------------------------------
 
-We highly recommend the following best practices, configuration options, and features for an optimal Mattermost user experience.
+You can customize which users can perform specific actions, such as creating teams, managing channels, posting reactions, `adding custom emojis <https://docs.mattermost.com/configure/configuration-settings.html#enable-custom-emoji>`__, or editing or deleting posts. See our `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation for details.
 
-**1. Upgrade your Mattermost server**
+Team creation
```
Comment (jasonblais) [inline source/getting-started/admin-onboarding-tasks.rst:89]: Is the intent for this section to recommend creating a public and private team? There isn't much detail in the team creation page about the use case for a public and private team.

File: source/getting-started/admin-onboarding-tasks.rst [hunk 2]
```diff
@@ -13,146 +13,202 @@ Administrator Onboarding Tasks
...
+1. Channels can be public and open, private and restrictive, direct with another user, direct with multiple users, or read-only. 
 
-**2. Install plugins**
+ - Convert channels from private to public and from public to private as needed. See our `Managing Channels <https://docs.mattermost.com/messaging/managing-channels.html>`__ documentation for details.
+ - Read-only channels are perfect for announcements because it’s easy to recall that information later. Update `channel moderation settings <https://docs.mattermost.com/onboard/advanced-permissions.html#read-only-channels-e20>`__ to set any channel as read-only.
 
-You can enable plugins and integrations to connect your team's workflows and toolsets into Mattermost. Plugins and integrations customize and extend the Mattermost platform.
+2. When creating any channel, we recommend using Markdown to populate the channel header with useful information and links relevant to all channel members, such as specifications, agendas, or other shared files. In addition, a soft channel naming convention helps users create and name new channels consistency, and find those channels easily later.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:102]: What is a "soft channel naming convention"? I think this would be a good place to link to that naming convention page in the docs! :)

File: source/getting-started/admin-onboarding-tasks.rst [hunk 3]
```diff
@@ -13,146 +13,202 @@ Administrator Onboarding Tasks
...
+| Reply notifications              | For deployments with Collapsed Reply Threads (Beta) disabled, each user can choose to receive notifications when someone replies to  |
+|                                  | a thread the user started or both started and participated in.                                                                       |
++----------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+
+| Automatic Direct Message Replies | Enable `Automatic Replies <https://docs.mattermost.com/configure/configuration-settings.html#enable-automatic-replies>`__            |
+|                                  | by going to **System Console > Experimental > Features** to allow all users to set an automated custom message that will             |
+|                                  | be sent once per day in response to Direct Messages.                                                                                 |
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:143]:
```suggestion
|                                  | be sent once per day in response to direct messages.                                                                                 |
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 4]
```diff
@@ -13,146 +13,202 @@ Administrator Onboarding Tasks
...
 
-  - Mattermost Enterprise customers can `enable ID-Only push notifications <https://docs.mattermost.com/configure/configuration-settings.html#push-notification-contents>`__ so push notification content is not passed through Apple Push Notification Service (APNS) or Google Firebase Cloud Messaging (FCM) before reaching the device. The ID-only push notification setting `offers a high level of privacy <https://mattermost.com/blog/id-only-push-notifications/>`__ while allowing team members to benefit from mobile push notifications.
+Visit the `Mattermost Marketplace <https://mattermost.com/marketplace/>`__ to learn about the many ways you can extend Mattermost functionality for your needs. To enable and manage plugins, go to **System Console > Plugins**. Then, download plugins from the Mattermost Marketplace.
 
-**5. Enable Custom Emoji**
+To enable integrations such as webhooks, slash commands, OAuth2.0, and bots, to go **System Console > Integrations**. See our `developer and integrator <https://developers.mattermost.com/integrate/other-integrations/>`__ for details.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:171]:
```suggestion
To enable integrations such as webhooks, slash commands, OAuth2.0, and bots, to go **System Console > Integrations**. See our `developer and integrator documentation <https://developers.mattermost.com/integrate/other-integrations/>`__ for details. 
```

**Review by justinegeffen — COMMENTED** [review]
Some minor feedback but otherwise this is a great improvement! :)

File: source/getting-started/admin-onboarding-tasks.rst (hunk 2 above)
Comment (cwarnermm) [reply source/getting-started/admin-onboarding-tasks.rst:140]: Great point! I've elected to remove some of this content because it has introduced unnecessary duplication.

File: source/getting-started/admin-onboarding-tasks.rst [hunk 5]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+- `Your workspace is accessible and performant <#review-mattermost-workspace-configuration>`__.
+- `The right people see and do the right things in Mattermost <#configure-user-permissions>`__.
+- `Users can sign in to Mattermost quickly and easily <#configure-user-authentication>`__.
 
- - Web server
- - Database
- - File storage
- - SMTP
- - Push Notification server
+We also encourage you to explore all of the settings available in the System Console. See our `Configuration Settings <https://docs.mattermost.com/configure/configuration-settings.html>`__ documentation for details on all supported options.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:24]:
```suggestion
We also encourage you to explore all of the settings available in the System Console. See our `configuration settings <https://docs.mattermost.com/configure/configuration-settings.html>`__ documentation for details on all supported options.
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 6]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+
+.. tip::
   
-These settings can also be set in the ``config.json`` file. Please see our `configuration settings documentation <https://docs.mattermost.com/configure/configuration-settings.html>`__ for a full listing of all configuration settings.
+  - Looking for more structure and guidance to plan out your Mattermost rollout? See our `Enterprise roll out checklist <https://docs.mattermost.com/getting-started/enterprise-roll-out-checklist.html>`__ documentation for details. 
+  - You can also manage Mattermost configuration settings in a ``config.json`` configuration file, located in the ``mattermost/config`` directory. See our `configuration settings <https://docs.mattermost.com/configure/configuration-settings.html>`__ documentation for details on ``config.json`` file equivalents.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:29]:
```suggestion
  - You can also manage Mattermost configuration settings in the ``config.json`` configuration file, located in the ``mattermost/config`` directory. See our `configuration settings <https://docs.mattermost.com/configure/configuration-settings.html>`__ documentation for details on ``config.json`` file equivalents.
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 7]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
++------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **Database**                 | Ensure that data management is configured for your needs. See the `Database <https://docs.mattermost.com/configure/configuration-settings.html#database>`__ documentation for details.              |
+|                              |                                                                                                                                                                                                     |
+|                              | If you have a `High Availability <https://docs.mattermost.com/scale/high-availability-cluster.html>`__  environment using PostgreSQL, we also recommend specific optimizations.                     |
+|                              | See the `High Availability Cluster recommendations <https://docs.mattermost.com/scale/high-availability-cluster.html#recommended-configuration-settings>`__ documentation for details.              |
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:46]:
```suggestion
|                              | See the `High Availability cluster recommendations <https://docs.mattermost.com/scale/high-availability-cluster.html#recommended-configuration-settings>`__ documentation for details.              |
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 8]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
++------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **File Storage**             | Ensure that file storage is configured for your needs. See the `File storage <https://docs.mattermost.com/configure/configuration-settings.html#file-storage>`__ documentation for details.         |
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:48]:
```suggestion
| **File Storage**             | Ensure that file storage is configured for your needs. See the `file storage <https://docs.mattermost.com/configure/configuration-settings.html#file-storage>`__ documentation for details.         |
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 9]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+| **SMTP**                     | Enable real-time Mattermost notifications for your users. See the `SMTP <https://docs.mattermost.com/configure/configuration-settings.html#smtp>`__ documentation for details.                      |
++------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **Push Notification Server** | Real-time `mobile push notifications <https://docs.mattermost.com/configure/configuration-settings.html#enable-push-notifications>`__ are enabled by default.                                       |
+|                              | You can control the `content included in notifications <https://docs.mattermost.com/configure/configuration-settings.html#push-notification-contents>`__.                                           |
+|                              | See the `Push Notification server <https://docs.mattermost.com/configure/configuration-settings.html#push-notification-server>`__ documentation for details.                                        |
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:54]:
```suggestion
|                              | See the `Push notification server <https://docs.mattermost.com/configure/configuration-settings.html#push-notification-server>`__ documentation for details.                                        |
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 10]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+| Go to...                       | To...                                                                                                                                                                                                 |
++================================+=======================================================================================================================================================================================================+
+| **Customization**              | Brand and customize how your users interact with your Mattermost workspace by:                                                                                                                        |
+|                                |                                                                                                                                                                                                       |
+|                                | - Customizing the `Support Email <https://docs.mattermost.com/configure/configuration-settings.html#support-email>`__ for email notifications, and onboarding tutorials, and support questions.       |
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:73]:
```suggestion
|                                | - Customizing the `support email <https://docs.mattermost.com/configure/configuration-settings.html#support-email>`__ for email notifications, onboarding tutorials, and support questions.       |
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 11]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+|                                | - Customizing the `Help Link <https://docs.mattermost.com/configure/configuration-settings.html#help-link>`__ to link to your help desk ticketing system.                                             |
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:74]:
```suggestion
|                                | - Customizing the `help link <https://docs.mattermost.com/configure/configuration-settings.html#help-link>`__ to link to your help desk ticketing system.                                             |
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 12]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+|                                | For additional security and protection with file attachments, a `ClamAV antivirus <https://mattermost.com/marketplace/antivirus-plugin/>`__                                                           |
+|                                | integration is available which scans files uploaded to Mattermost.                                                                                                                                    |
++--------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+3. From Mattermost v6.5, you can also review a dashboard of insights related to the health of your Mattermost workspace by going to **System Console > Reporting > Workplace Optimization**. See the `Optimize your Mattermost workspace <https://docs.mattermost.com/configure/optimize-your-workspace.html>`__ documentation for details.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:85]:
```suggestion
3. From Mattermost v6.5, you can also review a dashboard of insights related to the health of your Mattermost workspace by going to **System Console > Reporting > Workplace Optimization**. See the `optimize your Mattermost workspace <https://docs.mattermost.com/configure/optimize-your-workspace.html>`__ documentation for details. 
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 13]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+
+.. image:: ../images/workspace-optimization.png
+    :alt: Review a dashboard of insights related to the health of your Mattermost workspace.
+
+On the dashboard, you can see whether there's a Mattermost update you should install. Mattermost releases regular updates to `Mattermost Team Edition <https://mattermost.com/>`_ and `Mattermost Enterprise Edition <https://mattermost.com/pricing-self-managed/>`_. The `Mattermost Changelog <https://docs.mattermost.com/install/self-managed-changelog.html>`_ provides all information about changes in each version. When you upgrade your Mattermost server frequently, your users can access new features, improved user experiences, bug fixes, security fixes, and Mobile App compatibility. Upgrading your Mattermost server only takes a few minutes. See the `Upgrade Guide <https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html>`__ for step-by-step instructions.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:90]:
```suggestion
On the dashboard, you can see whether there's a Mattermost update you should install. Mattermost releases regular updates to `Mattermost Team Edition <https://mattermost.com/>`_ and `Mattermost Enterprise Edition <https://mattermost.com/pricing-self-managed/>`_. The `Mattermost changelog <https://docs.mattermost.com/install/self-managed-changelog.html>`_ provides all information about changes in each version. When you upgrade your Mattermost server frequently, your users can access new features, improved user experiences, bug fixes, security fixes, and Mobile App compatibility. Upgrading your Mattermost server only takes a few minutes. See the `upgrade guide <https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html>`__ for step-by-step instructions.
```

Comment (justinegeffen) [reply source/getting-started/admin-onboarding-tasks.rst:90]: Should we change the mentioned edition types?

File: source/getting-started/admin-onboarding-tasks.rst [hunk 14]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+
+Mattermost Enterprise customers can `enable Elasticsearch <https://docs.mattermost.com/scale/elasticsearch.html>`__ for optimized search performance at enterprise-scale. `Elasticsearch <https://docs.mattermost.com/scale/elasticsearch.html>`__ solves many known issues with full text database search, such as dots, dashes, and email addresses returning unexpected results. `Set up an Elasticsearch server <https://docs.mattermost.com/scale/elasticsearch.html#setting-up-an-elasticsearch-server>`__ and `enable Elasticsearch <https://docs.mattermost.com/configure/configuration-settings.html#elasticsearch>`__.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:92]:
```suggestion
Mattermost Enterprise customers can `enable Elasticsearch <https://docs.mattermost.com/scale/elasticsearch.html>`__ for optimized search performance at enterprise-scale. Elasticsearch solves many known issues with full text database search, such as dots, dashes, and email addresses returning unexpected results. `Set up an Elasticsearch server <https://docs.mattermost.com/scale/elasticsearch.html#setting-up-an-elasticsearch-server>`__ and `enable Elasticsearch <https://docs.mattermost.com/configure/configuration-settings.html#elasticsearch>`__.
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 15]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+---------------------------
+
+Once your Mattermost workspace is configured for your needs, focus next on ensuring the right people can see and do the right things in Mattermost, such as creating teams and managing channels, by controlling product access with `advanced permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__, `learning about teams <https://docs.mattermost.com/welcome/about-teams.html>`__, and `working with channels <https://docs.mattermost.com/guides/channels.html#work-with-channels>`__.
+
+.. image:: ../images/advanced-permissions.png
+    :alt: Control product access with granular Mattermost permissions.
+
+.. tip::
+
+  Mattermost won’t limit you to the number of teams you can create; however, a public team and an internal team are typically sufficient. See our `Creating Teams <https://docs.mattermost.com/welcome/about-teams.html#create-a-team>`__ and our `Team Settings <https://docs.mattermost.com/welcome/team-settings.html>`__ documentation for details.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:106]:
```suggestion
  Mattermost won’t limit you to the number of teams you can create; however, a public team and an internal team are typically sufficient. See our `creating teams <https://docs.mattermost.com/welcome/about-teams.html#create-a-team>`__ and our `team settings <https://docs.mattermost.com/welcome/team-settings.html>`__ documentation for details.
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 16]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+
+- Begin to onboard users by `enabling account creation <https://docs.mattermost.com/configure/configuration-settings.html#enable-account-creation>`__ or by connecting an authentication service to assist with user provisioning.
+
+- For bulk onboarding, enable `AD/LDAP Group Synchronization <https://docs.mattermost.com/configure/configuration-settings.html#enable-ad-ldap-group-sync>`__ to ensure new users are added to default teams and channels as they join Mattermost. See our `AD/LDAP Groups <https://docs.mattermost.com/onboard/ad-ldap-groups-synchronization.html>`__ documentation to learn more.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:117]:
```suggestion
- For bulk onboarding, enable `AD/LDAP group synchronization <https://docs.mattermost.com/configure/configuration-settings.html#enable-ad-ldap-group-sync>`__ to ensure new users are added to default teams and channels as they join Mattermost. See our `AD/LDAP groups <https://docs.mattermost.com/onboard/ad-ldap-groups-synchronization.html>`__ documentation to learn more.
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 17]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
+- Set personal reminders with `Remind <https://mattermost.com/marketplace/remind-plugin/>`__.
+- Create and share to do items with `Todo <https://github.com/mattermost/mattermost-plugin-todo>`__.
+- Customize welcome messages for new users with `WelcomeBot <https://mattermost.com/marketplace/welcomebot-plugin/>`__.
 
-If your organization requires more structure and project management artifacts for the implementation of Mattermost, please see our `Enterprise roll out checklist <https://docs.mattermost.com/getting-started/enterprise-roll-out-checklist.html>`__.
+Visit the `Mattermost Marketplace <https://mattermost.com/marketplace/>`__ to learn about the many ways you can extend Mattermost functionality for your needs. To enable and manage plugins, go to **System Console > Plugins**. Then, download plugins from the Mattermost Marketplace.
 
-Important Administration Notes 
-------------------------------
+To enable integrations such as webhooks, slash commands, OAuth2.0, and bots, to go **System Console > Integrations**. See our `developer and integrator documentation <https://developers.mattermost.com/integrate/other-integrations/>`__ for details.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:137]:
```suggestion
To enable integrations such as webhooks, slash commands, OAuth2.0, and bots, go to **System Console > Integrations**. See our `developer and integrations documentation <https://developers.mattermost.com/integrate/other-integrations/>`__ for details.
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 18]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
 - If you need to permanently delete a team or user, use the `mattermost user delete <https://docs.mattermost.com/manage/command-line-tools.html#mattermost-user-delete>`__ CLI command, or use the `mmctl user delete <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-user-delete>`__ command.
 
-Common Tasks
-------------
-
-**Creating System Admin account from the command line**
+Create a System Admin account from the command line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 - If the System Admin leaves the organization or is otherwise unavailable, you can use the command line interface to assign the *system_admin* role to an existing user. In the ``/opt/mattermost`` directory, type ``sudo -u mattermost bin/mattermost roles system_admin {user-name}``, where *{user-name}* is the username of the person with the new role. For more information about using the command line interface, see `Command Line Tools <https://docs.mattermost.com/manage/command-line-tools.html>`__.
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:150]:
```suggestion
- If the System Admin leaves the organization or is otherwise unavailable, you can use the command line interface to assign the *system_admin* role to an existing user. In the ``/opt/mattermost`` directory, type ``sudo -u mattermost bin/mattermost roles system_admin {user-name}``, where *{user-name}* is the username of the person with the new role. For more information about using the command line interface, see `command line tools <https://docs.mattermost.com/manage/command-line-tools.html>`__.
```

File: source/getting-started/admin-onboarding-tasks.rst [hunk 19]
```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
...
-- To preserve audit history, users are typically never deleted from the system. If permanently deleting a user is necessary (e.g. for the purposes of `GDPR <https://gdpr-info.eu/>`__), an `mmctl command <https://docs.mattermost.com/manage/mmctl-command-line-tool.html>`__ or a `CLI command <https://docs.mattermost.com/manage/command-line-tools.html>`_ can be used to do so.
-- Note that AD/LDAP user accounts cannot be deactivated from Mattermost; they must be deactivated from your Active Directory.
+If permanently deleting a user is necessary (e.g. for the purposes of `GDPR <https://gdpr-info.eu/>`__), an `mmctl command <https://docs.mattermost.com/manage/mmctl-command-line-tool.html>`__ or a `CLI command <https://docs.mattermost.com/manage/command-line-tools.html>`_ can be used to do so. AD/LDAP user accounts can't be deactivated from Mattermost; they must be deactivated from your Active Directory.
 
-**Checking for a valid license in Enterprise Edition without logging in**
+Check for a valid license in Enterprise Edition without logging in
```
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:166]: Should this be "an enterprise edition"?

**Review by justinegeffen — COMMENTED** [review]
Some formatting changes and minor content suggestions.

File: source/getting-started/admin-onboarding-tasks.rst (hunk 19 above)
Comment (cwarnermm) [reply source/getting-started/admin-onboarding-tasks.rst:166]: Good question! I wonder if this information applies equally to Professional as it does to Enterprise. If so, removing the plan details from the title and moving it to the description would work well.

Comment (justinegeffen) [reply source/getting-started/admin-onboarding-tasks.rst:166]: @cwarnermm, following up on this. :)

**Comment by cwarnermm** [comment]
This PR will eventually be closed in favour of https://github.com/mattermost/docs/pull/5664

**Comment by cwarnermm** [comment]
Closing this PR in favor of FY24 H1 initiatives that will help inform next steps on admin onboarding workflows.
